data/huion-kamvas-pro-19-gt1902.tablet

```diff
@@ -0,0 +1,28 @@
+# Huion
+# Kamvas Pro 19 (4K)
+# GT1902
+#
+# This tablet description file was initially generated using an automated tool:
+# https://github.com/JoseExposito/uclogic-libwacom-tablet-descriptions
+#
```

Comment:
```suggestion
```

you can drop this, it's only needed for the autogenerated and never-confirmed files.